ansible_collections/arista/avd/roles/eos_l3ls_evpn/README.md

```diff
@@ -1357,6 +1363,7 @@ port_profiles:
     flowcontrol:
       received: < received | send | on >
     qos_profile: < qos_profile_name >
+    macsec_profile: < macsec_profile_name >
```

Comment:
@xaviramon This is not added to any templates, so I have removed it from the rebase.